### ISSUE TITLE
Allow use without localisation dependencies

### ISSUE THREAD

**Issue by startswithaj** [siddii/angular-timer#191]
Version 1.2.0 didn't require Moment+locales or humanize-duration.

162K moment-with-locales.min.js
14K humanize-duration.js
---
176k just for deps (this is enormous for a single timer directive)

Moment alone is only 34K (moment.min.js). (Nearly the size of jquery)

Would it be difficult to have this directive check for those dependencies first and if they don't exist move on? I18n support is great in many use cases don't require it and 176k user for most that don't need it so could it be optional?

**Comment by siddii** [comment]
Yes, I've been wanting to do it. It doesn't make sense to have those dependencies for non-major use cases. I'll see if I can clean it up. 

Feel free to send a PR if you lend hands :)
